contracts/nftauction.js

```diff
@@ -180,15 +184,66 @@ const settleAuction = async (auction, index = null) => {
     // return the bids
     await returnBids(bids, priceSymbol);
 
-    // send payment to the seller
-    await api.transferTokens(seller, priceSymbol, leadBid.bid, 'user');
+    const token = await api.db.findOneInTable('tokens', 'tokens', { symbol: priceSymbol });
+    const price = api.BigNumber(leadBid.bid);
+    const feePercent = auction.fee / 10000;
+    let fee = price.multipliedBy(feePercent).decimalPlaces(token.precision);
+    if (fee.gt(price)) {
```

Comment:
Don't use this check, instead explicitly set the rounding mode to ROUND_DOWN

Reply:
This applies to all calls to decimalPlaces(), picking the rounding mode that makes sense for the operation